Thrive.csproj

```diff
@@ -33,11 +33,17 @@
     <Compile Include="src\engine\SettingValue.cs" />
     <Compile Include="src\engine\StartupActions.cs" />
     <Compile Include="src\general\PathUtils.cs" />
-    <Compile Include="src\general\EasterEggMessages.cs" />
     <Compile Include="src\general\HelpTexts.cs" />
     <Compile Include="src\general\HelpScreen.cs" />
     <Compile Include="src\general\PauseMenu.cs" />
+    <Compile Include="src\gui_common\tooltip\DefaultToolTip.cs" />
+    <Compile Include="src\gui_common\tooltip\ICustomToolTip.cs" />
+    <Compile Include="src\gui_common\tooltip\microbe_editor\ModifierInfoLabel.cs" />
+    <Compile Include="src\gui_common\tooltip\microbe_editor\SelectionMenuToolTip.cs" />
+    <Compile Include="src\gui_common\tooltip\ToolTipCallbackData.cs" />
+    <Compile Include="src\gui_common\tooltip\ToolTipManager.cs" />
     <Compile Include="src\gui_common\ControlHighlight.cs" />
+    <Compile Include="src\microbe_stage\editor\CollapsibleList.cs" />
```

Comment:
Wouldn't this collapsible list component be general enough to include in gui_common?

Reply:
It is, just not sure if other places will also use the style, but I'll move it

Reply:
They don't yet, but I foresee this being useful for all kinds of menus and future editors as well.